### DISCUSSION TITLE
[Feature Request] Sample with loader-less loading of Monaco Editor for the most basic usage

### DISCUSSION THREAD

**Discussion by vadimkantorov** [microsoft/monaco-editor#3803] [Q&A]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Description

Hi!

I'm building a rudimentary client-side, in-browser LaTeX compiler (TexLive transpiled with Emscripten) and editor (https://busytex.github.io).

The only dependency of this project is Monaco Editor (and I am not using any web tools such as npm/webpack - just one HTML file and five JS files). Currently, I'm loading Monaco Editor from unpkg.com CDN with an AMD loader. Ideally, I would like to get rid of the loader, host all Monaco JS/CSS files directly on my GitHub Pages and load all Monaco files directly with `<script>`/`<style>` tags. It would be great to have a sample for a basic usage like this at https://github.com/microsoft/monaco-editor/tree/main/samples (maybe adopt the sample from https://github.com/microsoft/monaco-editor/issues/2335#issuecomment-989078694 ?).

Maybe related/duplicate to https://github.com/microsoft/monaco-editor/issues/2335. But I'm not understanding client-side world enough to tell / understand properly various loaders technologies. Please feel free to close this if duplicate or point me to a sample already achieving this.

Thanks!

**Comment by vadimkantorov** [comment]
Currently, I ripped off all files in https://unpkg.com/browse/monaco-editor@0.34.1/min/vs/ and copied them over my website. It appears that there's few required files, but they are placed in several-layer directory structures :(

And it still would allow to kick-start hacking, if there was a variant that did not require a loader and it was possible to simply `<link>`/`<script type="module" src="...">` the few needed files

**Comment by rupurt** [comment]
I'd also love to use the monaco editor in a no build app :pray:
